clash-lib/clash-lib.cabal

```diff
@@ -279,6 +279,8 @@ Library
                       Clash.Util.Interpolate
                       Clash.Pretty
 
+                      Clash.Verification.PrettyPrinters
```

Comment:
`clash-lib` has some other modules just for pretty-printing which are called `Pretty` instead of `PrettyPrinters`. I think it would be good to rename this to be consistent